v7/config_role/config_role.py

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
An open-source license is mandatory for every plugin.

Reply:
Got it, I just added one